addons/website_sale/models/account_move.py

```diff
@@ -12,6 +13,13 @@ class AccountMove(models.Model):
         help='Website through which this invoice was created for eCommerce orders.',
         store=True, readonly=True, tracking=True)
 
+    def _auto_init(self):
+        if not column_exists(self.env.cr, "account_move", "website_id"):
+            # Because sale_order.website_id is a new field in this module,
+            # we can assume its value to be empty.
+            create_column(self.env.cr, "account_move", "website_id", "int4")
```

Comment:
Could you detail why a create column is needed here?
```suggestion
            # Create column is needed to avoid a memory error [...]
            # Because sale_order.website_id is a new field in this module,
            # we can assume its value to be empty.
            create_column(self.env.cr, "account_move", "website_id", "int4")
```